react/README.md

```diff
@@ -69,13 +69,13 @@
     **Component Naming**: Use the filename as the component name. For example, `ReservationCard.jsx` should have a reference name of `ReservationCard`. However, for root components of a directory, use `index.jsx` as the filename and use the directory name as the component name:
     ```javascript
     // bad
-    const Footer = require('./Footer/Footer.jsx')
+    import Footer from './Footer/Footer.jsx';
```

Comment:
can we omit the extension in all our examples? file extensions should ideally never be specified in a `require`